modules/admin_manual/nav.adoc

```diff
@@ -64,6 +64,7 @@
 **** Security
 ***** xref:configuration/server/security/password_policy.adoc[Password policy]
 ***** xref:configuration/server/security/oauth2.adoc[OAuth2]
+***** xref:configuration/server/security/hsmdaemon.adoc[The HSM Daemon]
```

Comment:
Not everyone is aware about the term HSM. I would therefore take the long name Hardware Security Module into the navigation and page header and then continue with the abbreviation HSM.